app/components/ui-kit/grid-item.js

```diff
@@ -25,15 +26,15 @@ function screenClass(key, screen) {
 
 const FLEX_SIZES = {
   INITIAL: 'initial',
-  EVEN: 'even',
+  FREE: 'free',
```

Comment:
I think using `one` or `1` for this could be weird or confusing, although it would be closest to the tailwind name. And I'm not sure that `even` is right either. This PR contains the first use of this utility, so now is a good time to decide what to call it.

Reply:
I'm not sure that I'm completely understanding the sense of the Tailwind property, but it looks like it just binds to `flex: <value>`. Is it correct? If so, then I'd like to suggest sticking with the CSS nature of the value, so that it was more intuitive. In this sense `even` sounds better to me. Or maybe `grow-single` (and eventually `grow-double`, `grow-triple`, etc.)? My concern about `free` is that most likely if I'd meet it somewhere in the code, I wouldn't understand what it's doing unless I look into implementation.

Reply:
Yeah that's pretty much correct, they are shorthands for `flex` property values. Here's the TW docs page for them https://tailwindcss.com/docs/flex . I agree, the naming is not really all that clear, so yeah maybe it would be good to replace with our own. So I've updated the tailwind config and the component to use such naming, let me know what you think.